src/compas_fab/data/robot_library/ur10e_robot/urdf/robot_description.urdf

```diff
@@ -0,0 +1,242 @@
+      <mass value="3.87"/>
+      <inertia ixx="0.11079302548902206" ixy="0.0" ixz="0.0" iyy="0.11079302548902206" iyz="0.0" izz="0.010884375"/>
+    </inertial>
+  </link>
+  <link name="wrist_1_link">
+    <visual>
+      <origin xyz="0.0 0.0 -0.135" rpy="1.5707963267948966 0.0 0.0"/>
+      <geometry>
+        <mesh filename="package://ur_description/meshes/ur10e/visual/wrist1.dae.obj"/>
+      </geometry>
+      <material name="LightGrey">
+        <color rgba="0.7 0.7 0.7 1.0"/>
+      </material>
+    </visual>
+    <collision>
+      <origin xyz="0.0 0.0 -0.135" rpy="1.5707963267948966 0.0 0.0"/>
+      <geometry>
+        <mesh filename="package://ur_description/meshes/ur10e/collision/wrist1.stl"/>
+      </geometry>
+    </collision>
+    <inertial>
+      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
+      <mass value="1.96"/>
+      <inertia ixx="0.005108247956699999" ixy="0.0" ixz="0.0" iyy="0.005108247956699999" iyz="0.0" izz="0.005512499999999999"/>
+    </inertial>
+  </link>
+  <link name="wrist_2_link">
+    <visual>
+      <origin xyz="0.0 0.0 -0.12" rpy="0.0 0.0 0.0"/>
+      <geometry>
+        <mesh filename="package://ur_description/meshes/ur10e/visual/wrist2.dae.obj"/>
+      </geometry>
+      <material name="LightGrey">
+        <color rgba="0.7 0.7 0.7 1.0"/>
+      </material>
+    </visual>
+    <collision>
+      <origin xyz="0.0 0.0 -0.12" rpy="0.0 0.0 0.0"/>
+      <geometry>
+        <mesh filename="package://ur_description/meshes/ur10e/collision/wrist2.stl"/>
+      </geometry>
+    </collision>
+    <inertial>
+      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
+      <mass value="1.96"/>
+      <inertia ixx="0.005108247956699999" ixy="0.0" ixz="0.0" iyy="0.005108247956699999" iyz="0.0" izz="0.005512499999999999"/>
+    </inertial>
+  </link>
+  <link name="wrist_3_link">
+    <visual>
+      <origin xyz="0.0 0.0 -0.1168" rpy="1.5707963267948966 0.0 0.0"/>
+      <geometry>
+        <mesh filename="package://ur_description/meshes/ur10e/visual/wrist3.dae.obj"/>
+      </geometry>
+      <material name="LightGrey">
+        <color rgba="0.7 0.7 0.7 1.0"/>
+      </material>
+    </visual>
+    <collision>
+      <origin xyz="0.0 0.0 -0.1168" rpy="1.5707963267948966 0.0 0.0"/>
+      <geometry>
+        <mesh filename="package://ur_description/meshes/ur10e/collision/wrist3.stl"/>
+      </geometry>
+    </collision>
+    <inertial>
+      <origin xyz="0.0 0.0 -0.025" rpy="0.0 0.0 0.0"/>
+      <mass value="0.202"/>
+      <inertia ixx="0.00014434577559500002" ixy="0.0" ixz="0.0" iyy="0.00014434577559500002" iyz="0.0" izz="0.00020452500000000002"/>
+    </inertial>
+  </link>
+  <link name="base"/>
+  <link name="flange"/>
+  <link name="tool0"/>
+  <joint name="base_link-base_link_inertia" type="fixed">
+    <parent link="base_link"/>
+    <child link="base_link_inertia"/>
+    <axis xyz="1.0 0.0 0.0"/>
+    <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 3.141592653589793"/>
```

Comment:
I've been considering this a lot, I really don't see any reason for why we should keep this rotation. It's just plain wrong in my opinion.

I asked about it on the ros-i repo, but there's no answer so far: https://github.com/ros-industrial/universal_robot/issues/676"

But considering the goal of this package is to simplify usage of robots, having this in place makes that objectively harder, so, I really think we should not include the rotation.

Reply:
As discussed, we will not manually modify the URDFs that are included in the RobotLibrary. The rationale is that these files are automatically generated from the ros_docker repository's included robot models. In the future, we can think about fixing the problem at that upper stream level.